atlasdb-client/src/main/java/com/palantir/atlasdb/AtlasDbConstants.java

```diff
@@ -30,7 +30,7 @@ private AtlasDbConstants() {
         // Utility
     }
 
-    public static final TableReference PUNCH_TABLE = TableReference.createWithEmptyNamespace("_punch");
+    public static final TableReference PUNCH_TABLE = TableReference.createWithEmptyNamespace("_punch2");
```

Comment:
Yep. The consequence of this would be that we would try to sweep `_punch`, which isn't very useful. (In previous versions this would cause sweep to explode - because we punch at commit timestamps, which don't have a transactions table entry - though we've made it more resilient since!)

Reply:
done